src/Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+<Project ToolsVersion="15.0">
+    <PropertyGroup>
+        <PublishRepositoryUrl>true</PublishRepositoryUrl>
+        <EmbedUntrackedSources>true</EmbedUntrackedSources>
```

Comment:
this embeds generate source files (like assemblyinfo) in the pdb for later extraction